9-regular-expressions/05-regexp-multiline-mode/article.md

```diff
@@ -1,14 +1,14 @@
-# Multiline mode of anchors ^ $, flag "m"
+# Modo multilinha das âncoras ^ e $, flag "m"
 
-The multiline mode is enabled by the flag `pattern:m`.
+O modo multilinha é ativado pela flag `pattern:m`.
```

Comment:
@Peruibeloko 


I normally translate the word `flag` in my other translations' work as `opção`, could you test it to see how it sounds?